nixos/tests/installer.nix

```diff
@@ -510,14 +510,8 @@ let
             ntp
             perlPackages.ListCompare
             perlPackages.XMLLibXML
-            python3Minimal
             # make-options-doc/default.nix
-            (let
-                self = (pkgs.python3Minimal.override {
-                  inherit self;
-                  includeSiteCustomize = true;
-                });
-              in self.withPackages (p: [ p.mistune ]))
+            python3.withPackages (p: [ p.mistune ])
```

Comment:
This broke eval of various installer tests, due to missing paranthesis.

Fixed in #280511.